src/main.rs

```diff
@@ -69,6 +73,15 @@ fn main() -> ! {
     wdt1.disable();
     {% endif -%}
 
+    {% if logging -%}
+    // setup logger
+    // To change the log_level change the env section in .config/cargo.toml
+    // or remove it and set ESP_LOGLEVEL manually before running cargo run
+    // this requires a clean rebuild because of https://github.com/rust-lang/cargo/issues/10358
```

Comment:
Maybe we could move this information to the README instead of having it on code?